docs/components/networkmanager.md

```diff
@@ -29,7 +28,6 @@ The `NetworkManager` is a required **Netcode for GameObjects (Netcode)** compone
 All `NetworkManager` sub-systems are instantiated once the `NetworkManager` is started (that is, `NetworkManager.IsListening == true`).  A good general "rule of thumb" is to not attempt to access the below sub-systems before starting the `NetworkManager`, otherwise they won't yet be initialized.
 :::
 
-- [NetworkManager.PrefabHandler](../advanced-topics/object-pooling.md): This provides access to the NetworkPrefabHandler that is used for NetworkObject pools and to have more control overriding network prefabs.
```

Comment:
PrefabHandler wasn't removed. Still around. From an API standpoint this was a non-breaking change. It's only a change to editor workflow.